mods/sp/weapons/superweapons.yaml

```diff
@@ -391,6 +413,10 @@ NuclearMissile:
 			DefenseArmor: 50
 			ConcreteArmor: 5
 		DamageTypes: Prone50Percent, TriggerProne, FireDeath, Incendiary
+	Warhead@GetsOneShotedbySuperweaponsAA: SpreadDamage
+		Spread: 1c512
+	Warhead@GetsOneShotedbySuperweapons: SpreadDamage
+		Spread: 1c512
```

Comment:
AoE might be a bit too big.

Reply:
how about 1c200? I am going to work so I cannot do test.